docs/build/micro-rdk/movement-sensor/gyro-mpu6050.md

```diff
@@ -6,7 +6,7 @@ type: "docs"
 description: "Configure an MPU-6050 movement sensor."
 images: ["/icons/components/imu.svg"]
 aliases:
-  - /micro-rdk/movement-sensor/gyro-mpu6050/
```

Comment:
@andf-viam @sguequierre why are you deleting previous aliases? Unless you are readding a page under that URL, previous URLs need to stay.